scripts/auto-rebase/rebase.sh

```diff
@@ -908,8 +903,8 @@ rebase_to() {
     local lvms_operator_bundle_manifest=$3
 
     title "# Rebasing to ${release_image_amd64} and ${release_image_arm64}"
-    download_release "${release_image_amd64}" "${release_image_arm64}"
-    download_lvms_operator_bundle_manifest "$lvms_operator_bundle_manifest"
+    #download_release "${release_image_amd64}" "${release_image_arm64}"
```

Comment:
I think some of these changes were probably for debugging?

Reply:
haha, yes :laughing: